spec/rails_helper.rb

```diff
@@ -1,4 +1,4 @@
-ENV['RAILS_ENV'] = 'test'
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
oh d'oh I changed this when debugging a weird issue -- my rails console thinks it is in test env by default. can't figure out why!